File: Data Structure II/python/Day 2/sort_colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        for i in range(len(nums)):
            min=i
            for j in range(i+1,len(nums)):
                if(nums[j]<nums[min]):
                  min=j
            
            if(min!=i):
                nums[min],nums[i]=nums[i],nums[min]

File: Data Structure II/Java/Day 2/SortColors.java
class Solution {
    public void sortColors(int[] nums) {
        for(int i=0;i<nums.length;i++){
            int min=i;
            for(int j=i+1;j<nums.length;j++){
                if(nums[j]<nums[min])
                min=j;
            }
            if(min!=i){
                int temp=nums[min];
                nums[min]=nums[i];
                nums[i]=temp;
            }
        }
    }
}